src/component/Label.tsx

```diff
@@ -509,16 +510,21 @@ const parseLabel = (label: any, viewBox: ViewBox) => {
   return null;
 };
 
-const renderCallByParent = (parentProps: any, viewBox?: ViewBox, checkPropsLabel = true) => {
+const renderCallByParent = (
+  parentProps: { children?: ReactNode; label?: unknown },
+  viewBox?: ViewBox,
+  checkPropsLabel = true,
+): ReactElement[] | null => {
   if (!parentProps || (!parentProps.children && checkPropsLabel && !parentProps.label)) {
     return null;
   }
   const { children } = parentProps;
   const parentViewBox = parseViewBox(parentProps);
 
-  const explicitChildren = findAllByType(children, Label).map((child, index: number) =>
+  const explicitChildren = findAllByType(children, Label).map((child, index) =>
     cloneElement(child, {
       viewBox: viewBox || parentViewBox,
+      // eslint-disable-next-line react/no-array-index-key
```

Comment:
Instead of ignoring eslint, how about following the rule instead? Does the child have a property from which we could generate a new key string?

Reply:
Due to the large use of `any` type it's difficult to find a suitable prop to use as a `key`. 
In this specific scenario I can't find a suitable prop to use.
If you have any idea let me know.

---

Since the warning are there from a certain amount if time and there are 9 additional disabled comment may I suggest you to consider to leave the disabled comment  and create a separate issue to address the issue in the entire codebase.

Otherwise I can remove the disable comment if you are ok with the rest of the code of the PR.

Reply:
Fair enough, please do create a follow up issue.